brain/neuron/morphology.cpp

```diff
@@ -89,7 +89,7 @@ const Vector2is& Morphology::getSections() const
     return *_impl->sections;
 }
 
-const SectionTypes& Morphology::getSectionTypes() const
+const brion::SectionTypes& Morphology::getSectionTypes() const
```

Comment:
I don't like that very much, because then you have brain::SectionTypes and brain::morphology::neuron::SectionTypes.